oracle/datadog_checks/oracle/oracle.py

```diff
@@ -53,6 +54,13 @@ class Oracle(AgentCheck):
         'Temp Space Used':                  'oracle.temp_space_used',
     }
 
+    PROCESS_METRICS = OrderedDict([
```

Comment:
Why use an ordered dict here?

Reply:
It's ordered because when the `SELECT ...` query runs, I need the columns to be in a specific order. I need the columns to be in a specific order because the process metrics are all on the same row:

PROGRAM | PGA_USED_MEM | PGA_ALLOC_MEM | PGA_FREEABLE_MEM | PGA_MAX_MEM
------------ | ------------- | ------------ | ------------- | -------------
PSEUDO | 1 | 2 | 3 | 4 |
PMON | 1 | 2 | 3 | 4 |

That means I have to iterate over the columns in a specific order.

I can technically not use an OrderedDict and instead use a list of tuples, but it's easier to get all the keys and values this way. I am not opposed to just using a list of tuples, however!

Reply:
Do we know they will always be in that order?

Reply:
We can rely on `SELECT ...` to maintain the column ordering :)

Reply:
okay, that's fine with me, I'll approve when you're happy with this

Reply:
Awesome, thank you :)